q/qemu-guest-agent.yml

```diff
@@ -0,0 +1,15 @@
+qemu-guest-agent:
+  image: ${REGISTRY_DOMAIN}/rancher/os-qemuguestagent:v1.3.0
+  command: ["/usr/bin/qemu-ga"]
+  privileged: true
+  restart: always
+  labels:
+    io.rancher.os.scope: system
+  pid: host
+  ipc: host
+  net: host
+  uts: host
+  volumes_from:
+  - command-volumes
```

Comment:
The `command-volumes` should be necessary.
Are `system-volumes` and `user-volumes` also needed?